runtime/stdlib/contracts/test.cdc

```diff
@@ -258,6 +290,24 @@ pub contract Test {
         /// Overrides any existing configuration.
         ///
         pub fun useConfiguration(_ configuration: Configuration)
+
+        /// Returns all the logs from the blockchain, up to the calling point.
+        ///
+        pub fun logs(): [String]
+
+        /// Returns the service account of the blockchain. Can be used to sign
+        /// transactions with this account.
+        ///
+        pub fun serviceAccount(): Account
+
+        /// Returns all events emitted from the blockchain, optionally filtered
+        /// by type name.
+        ///
+        pub fun events(_ typeName: String): [AnyStruct]
```

Comment:
Should this match the functions which got added above?

Reply:
I do not think that it's necessary. Relevant comment: https://github.com/onflow/cadence/pull/2505#discussion_r1206457859